main.typ

```diff
@@ -26,29 +26,44 @@
   // introducción, abstracto/resumen, ...
   = Agradecimientos
   #lorem(100)
+  #pagebreak()
+  #pagebreak()
```

Comment:
If you are trying to always land on an even/odd page, you can use the `to` parameter of `pagebreak`: https://typst.app/docs/reference/layout/pagebreak/

Reply:
No, I only wanted to  leave a 1 page gap between each heading. However Im going to change it, I think it would be better that way